cmd/nerdctl/ps.go

```diff
@@ -125,25 +130,39 @@ func printContainers(ctx context.Context, cmd *cobra.Command, containers []conta
 	if err != nil {
 		return err
 	}
+
 	var w io.Writer
 	w = os.Stdout
 	var tmpl *template.Template
 	format, err := cmd.Flags().GetString("format")
 	if err != nil {
 		return err
 	}
+
+	size := false
+	if !quiet {
+		size, err = cmd.Flags().GetBool("size")
+		if err != nil {
+			return err
+		}
+	}
+
 	switch format {
 	case "", "table":
 		w = tabwriter.NewWriter(os.Stdout, 4, 8, 4, ' ', 0)
 		if !quiet {
-			fmt.Fprintln(w, "CONTAINER ID\tIMAGE\tCOMMAND\tCREATED\tSTATUS\tPORTS\tNAMES")
+			printHeader := "CONTAINER ID\tIMAGE\tCOMMAND\tCREATED\tSTATUS\tPORTS\tNAMES"
+			if size {
+				printHeader += "\tSIZE"
+			}
+			fmt.Fprintln(w, printHeader)
 		}
 	case "raw":
 		return errors.New("unsupported format: \"raw\"")
 	case "wide":
 		w = tabwriter.NewWriter(os.Stdout, 4, 8, 4, ' ', 0)
 		if !quiet {
-			fmt.Fprintln(w, "CONTAINER ID\tIMAGE\tPLATFORM\tCOMMAND\tCREATED\tSTATUS\tPORTS\tNAMES\tRUNTIME")
+			fmt.Fprintln(w, "CONTAINER ID\tIMAGE\tPLATFORM\tCOMMAND\tCREATED\tSTATUS\tPORTS\tNAMES\tRUNTIME\tSIZE")
```

Comment:
Set size as true, and easy to handle later.

Reply:
Please see [previous comment(https://github.com/containerd/nerdctl/pull/1177#discussion_r910699012).

Reply:
Not just in the 221 lines, we can also optimize the logic after line 216, multi-layered `if` looks ugly.

@fahedouch WDYT?

Reply:
In addition, why not consider putting `Platform` at the end(in front of `Size`)?

- Consistent with the output of `nerdctl images`
- Most of the wide output is appended to the end

cc @AkihiroSuda

Reply:
The above can be another PR

Reply:
>  why not consider putting Platform at the end(in front of Size)?

Either is fine to me